price-server/src/app/mod.rs

```diff
@@ -77,12 +174,11 @@ impl PriceApp {
         &self,
         sats: Sats,
     ) -> Result<UsdCents, PriceAppError> {
-        let cents = self
-            .price_cache
-            .latest_tick()
-            .await?
-            .sell_usd()
-            .cents_from_sats(sats);
+        let cents = UsdCents::from_decimal(
+            self.price_mixer
+                .apply(|p| *p.sell_usd().cents_from_sats(sats.clone()).amount())
```

Comment:
I'm surprised `sats.clone()` is requires. `Decimal` implemented copy.

Reply:
It should be easy to add [here](https://github.com/GaloyMoney/stablesats-rs/blob/main/price-server/src/currency/mod.rs#L18)

Reply:
I can not implement Copy for the Currency-struct because the inner struct Money does not implement Copy. Any ideas on how to fix this?